### DISCUSSION TITLE
Accepting Payments in China

### DISCUSSION THREAD

**Discussion by burkybang** [Glench/ExtPay#106] [Q&A]
Someone from China is interested in purchasing a subscription to my extension. I guess Stripe isn't available there. Has anyone come up with a workaround for this?

**Comment by surtic86** [comment]
Accept it per something else and tell Glen to add him on the Account List for Access.

**Reply by FrazerHainsworth** [reply]
> Someone from China is interested in purchasing a subscription to my extension. I guess Stripe isn't available there. Has anyone come up with a workaround for this?

Stripe has Wechat pay available now so China can do

**Reply by burkybang** [reply]
@FrazerHainsworth Looking in my Stripe settings, it says WeChat Pay doesn't support recurring payments. My extension is subscription based.
![image](https://user-images.githubusercontent.com/927886/224163819-a0ddb124-38c0-41a8-822e-4bdae9bf2437.png)

@Glench Am I understanding this correctly?
